book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] you should also add a date for rental so it would be better if you directly create the rental here like the below example so that you will achieve the many-to-many relationships: 
```ruby
  def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```

Reply:
@youmari , I don't quite understand what you mean by this

Reply:
@Ibnballo1 For a rental to be created it needs date ,book, and person calling Rental.new and passing these 3 it will create a rental from the book using the `add_rental`